### PR TITLE
New plugin: capitalize

### PR DESCRIPTION
**Pull request by luximus-hunter**
Please let me know if i need to improve stuff 😓

### CODE REVIEW

File: src/plugins/capitalize.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+export default definePlugin({
+    name: "Remove Chat Buttons",
+    description: "Make every first letter of a sentence a capital letter.",
+    authors: [
+        {
+            id: 306692821358739456n,
+            name: "Luximus",
+        },
+    ],
+    patches: [],
```
Comment (MaiKokain) [inline src/plugins/capitalize.ts:35]: remove if not being used

Comment (luximus-hunter) [reply src/plugins/capitalize.ts]: My bad!, I removed it

File: src/plugins/capitalize.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import definePlugin, { OptionType } from "../utils/types";
+import { addPreEditListener, addPreSendListener, MessageObject, removePreEditListener, removePreSendListener } from "../api/MessageEvents";
+import { Settings } from "../Vencord";
+
+const blacklistSeperator = ";";
+const urlRegex = new RegExp("^((http[s]?|ftp):\/)?\/?([^:\/\s]+)((\/\w+)*\/)([\w\-\.]+[^#?\s]+)(.*)?(#[\w\-]+)?$");
+
+export default definePlugin({
+    name: "Remove Chat Buttons",
+    description: "Make every first letter of a sentence a capital letter.",
```
Comment (taskylizard) [inline src/plugins/capitalize.ts:28]: i don't think this is intentional

**Comment by Vendicated** [comment]
Thanks! But to be honest, this is pretty useless. Why Would Anyone Want To Talk Like This?

You can post it in the third party plugins channel if you want though

**Comment by luximus-hunter** [comment]
> Why Would Anyone Want To Talk Like This?

It doesn't capitalize each word in a sentence. It just does the first word of it.

**Comment by Vendicated** [comment]
Oh l see, my bad. In that case however it should be part of the KyzaMode plugin that already has an open PR

**Comment by Vendicated** [comment]
#29

**Comment by luximus-hunter** [comment]
Alright! I didn't see it as it has a name that doesn't suggest it 😓 

Thanks either way tho!

**Pull request closed without merging**